src/traces/sankey/plot.js

```diff
@@ -124,7 +124,8 @@ module.exports = function plot(gd, calcData) {
     };
 
     var linkHoverFollow = function(element, d) {
-
+        var trace = gd._fullData[d.traceId];
+        var ptNumber = d.originalIndex;
```

Comment:
@monfera does this look ok to you? I'm looking for the data array index corresponding to this link / node (below).

Oh wait. `node` and `link` can have different lengths, correct? So maybe `hoverlabel` should be set per-node and per-link e.g.:

```js
var trace = {
  node: {
     hoverlabel: {
        bgcolor: [/* same length as node data arrays */]
     },
     link: {
       hoverlabel: {
         bgcolor: [/* same length as link data arrays */]
       }
     }
   },
   hoverlabel: {
     // for trace-wide hover label settings (that would be applied to all nodes and links)
   }
}
```

Well the above sounds uselessly complicated for v1 of sankey. So, I'd vote for not allowing array values in `hoverlabel` attribute of sankey traces. Does that sound ok?